src/utils/index.js

```diff
@@ -1,7 +1,15 @@
-import { filterProducts } from './filterProducts';
-import { sortProducts } from './sortProducts';
-import { rateProducts } from './rateProducts';
-import { priceFilter } from './priceFilter';
-import { filterCategory } from './filterCategory';
+import { filterProducts } from "./filterProducts";
+import { sortProducts } from "./sortProducts";
+import { rateProducts } from "./rateProducts";
+import { priceFilter } from "./priceFilter";
+import { filterCategory } from "./filterCategory";
+import { isItemInWishlist } from "./isItemInWishlist";
```

Comment:
You need not import and export. You can write `export { function/component } from path` and it will work in the same way.

Reply:
Actually, just for better understanding, I have kept it like this. But I Will improve it. thanks for suggestion